### PR TITLE
fix(group): never get defocused event

### PR DESCRIPTION
**Pull request by axos88**
Fixes an edge case when removing an already focused event that is the only focusable in the group, it never gets the defocused event

### CODE REVIEW

File: tests/src/test_cases/test_group.c [hunk 1]
```diff
@@ -58,4 +58,162 @@ void test_group_obj_by_index(void)
...
+{
+    uint32_t * counter = lv_event_get_user_data(e);
+    (*counter)++;
+}
+
+static void count_focused_cb(lv_event_t * e)
+{
+    uint32_t * counter = lv_event_get_user_data(e);
+    (*counter)++;
+}
```
Comment (AndreCostaaa) [inline tests/src/test_cases/test_group.c]: Please rebase and when doing so, make this a single `count_event_cb` function

**Review by AndreCostaaa — APPROVED** [review]
